charts/stable/sonarr/templates/_prometheusrules.tpl

```diff
@@ -0,0 +1,23 @@
+{{- define "uptime.prometheusrule" -}}
+{{- if hasKey .Values "metrics" }}
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "tc.common.names.fullname" . }}
+  labels:
+    {{- include "tc.common.labels" . | nindent 4 }}
+    {{- with .Values.metrics.prometheusRule.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  groups:
+    - name: {{ include "tc.common.names.fullname" . }}
+      rules:
+        {{- with .Values.metrics.prometheusRule.rules }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+{{- end }}
+{{- end }}
+{{- end -}}
```

Comment:
No, all metrics is handeled by common now.
These files are not allowed anymore.

Use the `metrics` section of values.yaml instead.